docs/can-guides/commitment/migrating_to_3.0.md

```diff
@@ -244,7 +244,7 @@ Instead, the first argument to [can-construct.extend] is the name of the constru
 If you use StealJS, you’ll need to install [steal-stache] to load your templates:
 
 ```
-npm install steal-stache --save
+npm install steal-stache@3 --save
```

Comment:
maybe if there becomes a can 4.0 and you still want the migrating to 3.0 guide to work.